### PR TITLE
Add OSPO docs and related policies

### LINKED ISSUE #3: Add OSPO docs and related policies
ℹ️ This issue is part of Epic #1  

## Objective
Ensure the project contains an updated version of the following files.

## Checklist
- [x]  README
- [x]  LICENSE
- [x]  CODE OF CONDUCT
- [x]  CONTRIBUTING
- [x]  CHANGELOG
- [x]  SECURITY DISCLOSURE
- [x]  GITHUB FUNDING

### PR DESCRIPTION
**Pull request by boterop**
## Description

Fixes #3 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [x] This change is a documentation update

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] ~~I have commented on my code, particularly in hard-to-understand areas~~
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] ~~I have added tests that prove my fix is effective or that my feature works~~
- [ ] ~~New and existing unit tests pass locally with my changes~~

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,9 +14,16 @@ def deps do
...
 ```
 
-Documentation can be generated with [ExDoc](https://github.com/elixir-lang/ex_doc)
-and published on [HexDocs](https://hexdocs.pm). Once published, the docs can
-be found at [https://hexdocs.pm/kadena](https://hexdocs.pm/kadena).
+## Configuration
+```elixir
+config :kadena, network: :test # Default is `:test`. To use the public network, set it to `:public`
+
+```
```
Comment (juanhurtado10) [inline README.md]: @santiagoboterokommit why add this config without an implementation?

File: README.md [hunk 2]
```diff
@@ -15,9 +14,16 @@ def deps do
 end
 ```
 
-Documentation can be generated with [ExDoc](https://github.com/elixir-lang/ex_doc)
-and published on [HexDocs](https://hexdocs.pm). Once published, the docs can
-be found at [https://hexdocs.pm/kadena](https://hexdocs.pm/kadena).
+## Configuration
+```elixir
+config :kadena, network: :test # Default is `:test`. To use the public network, set it to `:public`
+
```
Comment (juanhurtado10) [inline README.md]: We have not yet defined how to manage Kadena's network configuration. This is an assumption

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -43,14 +58,20 @@ alternatives.
 
 Get in touch: [oss@kommit.co][mail-to] | [@kommitters_oss][twitter] on Twitter.
 
+## Acknowledgements
+
+This document is inspired by:
+* https://github.com/kommitters/stellar_sdk/blob/main/CONTRIBUTING.md
```
Comment (juanhurtado10) [inline CONTRIBUTING.md]: @santiagoboterokommit this is not really required, we are under the same OSPO

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,33 +1,48 @@
-# Contributing 🎉
```
Comment (juanhurtado10) [inline CONTRIBUTING.md:1]: @santiagoboterokommit Please, do not change this headline

**Review by juanhurtado10 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (boterop) [reply README.md]: The [next PR](https://github.com/kommitters/kadena/pull/7) will add a config folder with that lines :thinking:

Comment (juanhurtado10) [reply README.md]: @santiagoboterokommit it should be added in the next PR then

**Review by juanhurtado10 — APPROVED** [review]

**Pull request merged**